tests/framework/InternalParallelTests/test_internal_parallel_ROM_scikit.xml

```diff
@@ -15,12 +15,14 @@
       <revision author="maljdan" date="2016-04-25">Switching to a different regressor that yields more consistent results.</revision>
       <revision author="maljdan" date="2016-04-25">Regolding based on 1-nearest neighbor regression. This should effectively report the same information as the training data.</revision>
```

Comment:
Since the HybridModel is a new Model entity, Cluster tests should be added as well (using internal Models (external models) and Codes)